File: nukkit_binding/src/main/java/com/blackjack200/xyron/nukkit/XyronData.java
package com.blackjack200.xyron.nukkit;

import com.github.blackjack200.xyron.Xchange;
import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
public class XyronData {
    @Getter
    private Xchange.PlayerReceipt receipt;
    @Getter
    private BufferedDataQueue queue;
}

File: pmmp_binding/src/prokits/xyron/loader/XyronData.php
<?php

namespace prokits\xyron\loader;

use prokits\xyron\PlayerReceipt;

class XyronData {
	public function __construct(
		public PlayerReceipt     $receipt,
		public BufferedDataQueue $queue,
	) {
	}

	public function getQueue() : BufferedDataQueue { return $this->queue; }
}